plugins/win-wasapi/win-wasapi.cpp

```diff
@@ -1506,12 +1514,33 @@ static obs_properties_t *GetWASAPIPropertiesInput(void *)
 					     device.id.c_str());
 	}
 
-	obs_properties_add_bool(props, OPT_USE_DEVICE_TIMING,
-				obs_module_text("UseDeviceTiming"));
+	obs_property_t *device_timing_prop =
+		obs_properties_add_bool(props, OPT_USE_DEVICE_TIMING,
+					obs_module_text("UseDeviceTiming"));
+
+	obs_properties_add_bool(props, OPT_ASYNC_COMPENSATION,
+				obs_module_text("AsyncCompensation"));
+
+	obs_property_set_modified_callback(device_timing_prop,
+					   device_timing_changed);
```

Comment:
'device_timing_changed': undeclared identifier

Reply:
The compile error is resolved.